PdfScribe/App.config

```diff
@@ -26,19 +26,18 @@
 	</system.diagnostics>
 	<applicationSettings>
 		<PdfScribe.Properties.Settings>
-			<setting name="AskUserForOutputFilename" serializeAs="String">
-				<value>True</value>
-			</setting>
-			<setting name="OpenAfterCreating" serializeAs="String">
-				<value>False</value>
-			</setting>
-			<setting name="OutputFile" serializeAs="String">
-				<value>%UserProfile%\PDFSCRIBE.PDF</value>
-			</setting>
-			<setting name="StripNoRedistill" serializeAs="String">
-				<!-- This will slow things down because another pass will be made through the postscript file - set it to "False" unless needed -->
-				<value>True</value>
-			</setting>
-		</PdfScribe.Properties.Settings>
+   <setting name="OutputFile" serializeAs="String">
+    <value>%UserProfile%\PDFSCRIBE.PDF</value>

```

Comment:
This should probably change to AIVIKA.PDF

Reply:
noted.